kong/plugins/response-ratelimiting/header_filter.lua

```diff
@@ -14,6 +14,14 @@ local _M = {}
 local function parse_header(header_value, limits)
   local increments = {}
   if header_value then
+    if (type(header_value) == "table") then
+      -- multiple headers with same name: combine per RFC 2616 section 4.2
+      local hval_str = header_value[1]
+      for i = 2,#header_value do
+        hval_str = hval_str .. "," .. header_value[i]
+      end
+      header_value = hval_str
+    end
```

Comment:
Hello @jdmonin, thx for the contribution. I believe that you changes can be replaced with a more concise version:

```lua
if type(header_value) == "table" then
  header_value = table.concat(header_value, ",")
end
```

And that would be it.